client/src/components/Nav/Nav.css

```diff
@@ -0,0 +1,133 @@
+* {

```

Comment:
This will affect all elements on the page, not just the navbar. Are you sure this is what you want?